helix-front/server/controllers/d.ts

```diff
@@ -1,6 +1,7 @@
 import { Request } from 'express';
+import request from 'request';
 
-export interface HelixUserRequest extends Request {
+export interface HelixRequest extends Request {
```

Comment:
Is there a reason for changing interface name here? Could this change break backward compatibility?

Reply:
This makes sense. Thanks